### ISSUE TITLE
Several errors when validate url that doesn't appear with code

### ISSUE THREAD

**Issue by BRONSON77** [validator/validator#820]
I'm trying to validate this url: http://barardino.vzpla.net/

These are some errors and warnings the validator has found:
-  Consider adding a lang attribute to the html start tag to declare the language of this document
- The character encoding was not declared. Proceeding using windows-1252
- Start tag seen without seeing a doctype first. Expected `<!DOCTYPE html>`
- Element head is missing a required instance of child element title

The problem is that my code already has lang, encoding, doctype and title declared. If I paste the code instead of insert the URI these problems doesn't appear. Any idea?

Thanks in advance.

**Comment by nschonni** [comment]
The validator is loading the site without javascript, all those errrors are correct when I tried to load your site with JavaScript disabled.

**Comment by BRONSON77** [comment]
So is there a problem with my JavaScript code?

**Comment by nschonni** [comment]
I think it's because those are the issues that are seen by your site if JavaScript isn't running.
EX: running `curl -H "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.89 Safari/537.36" http://barardino.vzpla.net/` to pretend to be Chrome gives
```html
<html>

<body>
    <script type="text/javascript" src="/aes.js"></script>
    <script>function toNumbers(d) { var e = []; d.replace(/(..)/g, function (d) { e.push(parseInt(d, 16)) }); return e } function toHex() { for (var d = [], d = 1 == arguments.length && arguments[0].constructor == Array ? arguments[0] : arguments, e = "", f = 0; f < d.length; f++)e += (16 > d[f] ? "0" : "") + d[f].toString(16); return e.toLowerCase() } var a = toNumbers("f655ba9d09a112d4968c63579db590b4"), b = toNumbers("98344c2eee86c3994890592585b49f80"), c = toNumbers("ff17036e856cdf16c4b475093a626b3b"); document.cookie = "__test=" + toHex(slowAES.decrypt(c, 2, a, b)) + "; expires=Thu, 31-Dec-37 23:55:55 GMT; path=/"; location.href = "http://barardino.vzpla.net/?i=1";</script>
    <noscript>This site requires Javascript to work, please enable Javascript in your browser or use a browser with
        Javascript support</noscript></body>

</html>
```
Which are why you see those issues being flagged.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/820#issuecomment-496081297
